src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ShopService {
+    public Storage process(List<FruitTransaction> transactions);
```

Comment:
The return type of `process` method should not be `Storage`. According to the SOLID principles, specifically the Dependency Inversion Principle, higher-level modules should not depend on lower-level modules, but they should depend on abstractions. Instead of returning `Storage`, consider returning a result that represents the outcome of processing the transactions, which could be a report or a status object.